### DISCUSSION TITLE
Possibility to set base_path?

### DISCUSSION THREAD

**Discussion by bijucyborg** [FlowiseAI/Flowise#532] [Q&A]
I'm trying to create a jupyter server proxy [extension](https://github.com/bijucyborg/jupyter_flowise_proxy) for Flowise. If successful, users could launch flowise from their jupyter lab or jupyter hub environments.

The problem is that the application uses absolute paths instead of relative paths.
script defer=“defer” src=“/static/js/main.39061e83.js”

So when the jupyter proxy extension tries to load the files it refers to the wrong path.

This can be overcome if there was a way to set a base path for the web app as a command line switch. 

Is there already a way to set the base path for the app?

Thanks in advance for any suggestions.

**Comment by HenryHengZJ** [comment]
perhaps this is where you can try changing - https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/index.ts#L561

**Comment by jeme95** [comment]
@bijucyborg 
did you make progress with it? or did you somehow manage to do it?

@HenryHengZJ 
the referenced line doesn't line up, could you shortly describe, what to change?
